communitiesuk/funding-service-design-post-award-prototype-api
# Mock db dataclasses

### Link to ticket
[PAB-5](https://trello.com/c/U1RCB79N/5-create-project-package-contact-organisation-models-and-apis)

### Change description
- Adds 4 sheets of data in dictionary format from the data model excel document as mocked data
- Implements some initial data classes to represent the mocked data
- Implements a `FakeDB` to store collections of data classes in
- Implements `from_dict` methods that parse the mock dictionary into a FakeFB full of dataclasses



### How to test

run:
`from db import db`
in a python console

This will run the `__init__.py` module in `db` and initialize and import a `FakeDB`

You could then interact with it in the python console to check it seems sensible etc...

### Screenshots of UI changes (if applicable)
